hyperledger-fabric/README.md

```diff
@@ -0,0 +1,16 @@
+# Flogo Enterprise extensions for Hyperledger Fabric
+
+This package includes [TIBCO Flogo® Enterprise](https://docs.tibco.com/products/tibco-flogo-enterprise-2-6-1) extensions for developing blockchain apps and smart contracts on [Hyperledger Fabric](https://www.hyperledger.org/projects/fabric).
```

Comment:
I think we should keep smart contracts and client app separate how about this text:

This package includes [TIBCO Flogo® Enterprise](https://docs.tibco.com/products/tibco-flogo-enterprise-2-6-1) extensions for developing blockchain client apps and [Dovetail](https://tibcosoftware.github.com/dovetail) extensions for developing blockchain smart contracts on [Hyperledger Fabric](https://www.hyperledger.org/projects/fabric).